sqlglot/dialects/postgres.py

```diff
@@ -248,11 +248,10 @@ class Postgres(Dialect):
     }
 
     class Tokenizer(tokens.Tokenizer):
-        QUOTES = ["'", "$$"]
```

Comment:
this was incorrect before?

Reply:
more like partially correct; it didn't take into account tagged heredocs like the following:

```
georgesittas=# select $foo$this is a string$foo$;
     ?column?
------------------
 this is a string
(1 row)

Reply:
FYI the relevant commit is https://github.com/tobymao/sqlglot/pull/617/files, cc @mpf82